# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1372492999 -7200
#      Sat Jun 29 10:03:19 2013 +0200
# Node ID a472bfc67b6c9c2b64f889b22122b00298a36d9d
# Parent  a86327a7d9fb28e06081b8355ffed685117f7032
fix of typos in language files

* es_ES.ts, fr_FR.ts, nl_NL.ts, pt_BR.ts

diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -470,17 +470,17 @@ has been deleted or renamed. Do you want
     <message>
         <location line="+25"/>
         <location line="+11"/>
         <source>Delete file/directory</source>
         <translation>Eliminar archivo/directorio</translation>
     </message>
     <message>
         <location line="-10"/>
-        <source>Are you sre you want to delete
+        <source>Are you sure you want to delete
 </source>
         <translation>¿Está seguro que desea eliminar\n</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
         <translation>No se puede eliminar un directorio que no esté vacio</translation>
     </message>
@@ -1087,22 +1087,16 @@ has been deleted or renamed. Do you want
         <translation>Ocultar widget</translation>
     </message>
     <message>
         <location line="+82"/>
         <source>Dock widget</source>
         <translatorcomment>Uso &quot;widget&quot; por ser el término usual: http://es.wikipedia.org/wiki/Widget</translatorcomment>
         <translation>Acoplar widget</translation>
     </message>
-    <message>
-        <location line="+29"/>
-        <source>Unock widget</source>
-        <translatorcomment>Uso &quot;widget&quot; por ser el término usual: http://es.wikipedia.org/wiki/Widget</translatorcomment>
-        <translation>Desacoplar widget</translation>
-    </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+85"/>
         <source>help</source>
         <translation>Ayuda</translation>
     </message>
@@ -1161,17 +1155,17 @@ has been deleted or renamed. Do you want
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
         <translation>Íconos de letras</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>Graphic  icons</source>
+        <source>Graphic icons</source>
         <translation>Íconos gráficos</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
@@ -1438,17 +1432,17 @@ has been deleted or renamed. Do you want
         <location filename="../src/settings-dialog.cc" line="+69"/>
         <location line="+4"/>
         <location line="+334"/>
         <source>System setting</source>
         <translation>Configuración del sistema</translation>
     </message>
     <message>
         <location line="-137"/>
-        <source>Difference to the defalt size</source>
+        <source>Difference to the default size</source>
         <translation>Diferencia con el tamaño predeterminado</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
         <translation>Color de fondo, rosa (255,0,255) significa predeterminado</translation>
     </message>
     <message>
diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -531,17 +531,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     <message>
         <location line="+25"/>
         <location line="+11"/>
         <source>Delete file/directory</source>
         <translation>Supprimer fichier/répertoire</translation>
     </message>
     <message>
         <location line="-10"/>
-        <source>Are you sre you want to delete
+        <source>Are you sure you want to delete
 </source>
         <translation>Etes-vous sûr de vouloir supprimer
 </translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
         <translation>Impossible de supprimer un répertoire qui n&apos;est pas vide</translation>
@@ -1146,21 +1146,16 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <source>Hide widget</source>
         <translation>Cacher le widget</translation>
     </message>
     <message>
         <location line="+82"/>
         <source>Dock widget</source>
         <translation>Attacher le widget</translation>
     </message>
-    <message>
-        <location line="+29"/>
-        <source>Unock widget</source>
-        <translation>Détacher le widget</translation>
-    </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+85"/>
         <source>help</source>
         <translation>aide</translation>
     </message>
@@ -1217,17 +1212,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
         <translation>Icones textuelles</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>Graphic  icons</source>
+        <source>Graphic icons</source>
         <translation>Icones graphiques</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Editor</source>
         <translation>Editeur</translation>
     </message>
     <message>
@@ -1509,17 +1504,17 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+129"/>
-        <source>Difference to the defalt size</source>
+        <source>Difference to the default size</source>
         <translation>Difference avecla taille par défaut</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
         <translation>Couleur de fond, rose [255,0,255] designe la valeur par défaut</translation>
     </message>
     <message>
diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -1121,17 +1121,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+25"/>
         <location line="+11"/>
         <source>Delete file/directory</source>
         <translation type="unfinished">Wis bestand/map</translation>
     </message>
     <message>
         <location line="-10"/>
-        <source>Are you sre you want to delete
+        <source>Are you sure you want to delete
 </source>
         <translation type="unfinished">Weet u zeker dat u dit wilt verwijderen
         </translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
         <translation type="unfinished">Kan geen niet-lege map wissen</translation>
@@ -1736,21 +1736,16 @@ is gewist of hernoemd. Wil je het nu ops
         <source>Hide widget</source>
         <translation type="unfinished">Widget verbergen</translation>
     </message>
     <message>
         <location line="+82"/>
         <source>Dock widget</source>
         <translation type="unfinished">Widget in venster opnemen</translation>
     </message>
-    <message>
-        <location line="+29"/>
-        <source>Unock widget</source>
-        <translation type="unfinished">Laat widget zweven</translation>
-    </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+85"/>
         <source>help</source>
         <translation type="unfinished">hulp</translation>
     </message>
@@ -1807,17 +1802,17 @@ is gewist of hernoemd. Wil je het nu ops
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
         <translation>Letter iconen</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>Graphic  icons</source>
+        <source>Graphic icons</source>
         <translation>Grafische iconen</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
@@ -2099,17 +2094,17 @@ is gewist of hernoemd. Wil je het nu ops
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
         <translation type="unfinished">Liggend streepje</translation>
     </message>
     <message>
         <location line="+129"/>
-        <source>Difference to the defalt size</source>
+        <source>Difference to the default size</source>
         <translation type="unfinished">Verschil met standaardgrootte</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
         <translation type="unfinished">Achtergrondkleur; roze (255,0,255) staat voor standaardkleur</translation>
     </message>
     <message>
diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_BR.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_BR.ts
@@ -527,17 +527,17 @@ para:</translation>
     <message>
         <location line="+25"/>
         <location line="+11"/>
         <source>Delete file/directory</source>
         <translation>Remover arquivo/diretório</translation>
     </message>
     <message>
         <location line="-10"/>
-        <source>Are you sre you want to delete
+        <source>Are you sure you want to delete
 </source>
         <translation>Você tem certeza que deseja remover</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Can not delete a directory that is not empty</source>
         <translation>Não é possível remover um diretório que não está vázio</translation>
     </message>
@@ -1139,21 +1139,16 @@ para:</translation>
         <source>Hide widget</source>
         <translation>Esconder widget</translation>
     </message>
     <message>
         <location line="+82"/>
         <source>Dock widget</source>
         <translation>Acoplar widget</translation>
     </message>
-    <message>
-        <location line="+29"/>
-        <source>Unock widget</source>
-        <translation>Desacoplar widget</translation>
-    </message>
 </context>
 <context>
     <name>octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+85"/>
         <source>help</source>
         <translation>ajuda</translation>
     </message>
@@ -1210,17 +1205,17 @@ para:</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
         <translation>Ícones de letra</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>Graphic  icons</source>
+        <source>Graphic icons</source>
         <translation>Ícones de gráficos</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
@@ -1502,17 +1497,17 @@ para:</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Underline Cursor</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+129"/>
-        <source>Difference to the defalt size</source>
+        <source>Difference to the default size</source>
         <translation>Diferença do tamanho padrão</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
         <translation>Cor de fundo, rosa (255,0,255) significa padrão</translation>
     </message>
     <message>
